lib/themes/dosomething/paraneue_dosomething/js/app.js

```diff
@@ -42,6 +42,11 @@ import 'validators/organDonation';
 
 import 'campaign/prototype';
 
+// Load React components
+import 'components/onboard';
```

Comment:
Let's move these to a separate entry point (that we conditionally load via the feature flag), so we're not loading a bunch of code that isn't getting used for every user. You can add a new entry-point in the [webpack config](https://github.com/DoSomething/phoenix/blob/dba088f6e7c1bc48e830019a4dfd5fcccdca9d33/lib/themes/dosomething/paraneue_dosomething/webpack.config.js#L6-L7), and then add it to the action page via `drupal_add_js`.